src/definitions/modules/tab.js

```diff
@@ -679,6 +684,29 @@ $.fn.tab = function(parameters) {
           }
         },
 
+        determine: {
+          activeTab: function() {
+            var activeTab = null;
+
+            $tabs.each(function(_index, tab) {
```

Comment:
`Array.forEach` is in efficient for this purpose since it iterate all elements (tabs).
We can use `Array.find` to find first element that satisy predicate when we drop IE support.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

For now, we can use `Array.some` that can stop iteration when first satisfying element found.

```javascript
var activeTab = null;
$tabs.some(function(_index, tab) {
    var $tab = $(tab);
    if( $tab.hasClass(className.active) ) {
        var
            tabPath = $(this).data(metadata.tab),
            $anchor = $allModules.filter('[data-' + metadata.tab + '="' + module.escape.string(tabPath) + '"]')
        ;

        if( $anchor.hasClass(className.active) ) {
            activeTab = tabPath;
            return true;
        }
    }
    return false;
});
```

Reply:
Using the `some` function will make me convert my `$tabs` to an array, which is not wanted since I'll need to get the tab object by another way. I think that actually it's not the best option to use, but it's the most convenient.